sql/ddl_api.sql

```diff
@@ -92,26 +92,19 @@ CREATE OR REPLACE FUNCTION  set_number_partitions(
 -- Drop chunks older than the given timestamp for the specific
 -- hypertable or continuous aggregate.
 CREATE OR REPLACE FUNCTION drop_chunks(
-    hypertable_or_cagg  REGCLASS,
-    older_than "any" = NULL,
-    newer_than "any" = NULL,
-    verbose BOOLEAN = FALSE
+    hypertable_or_cagg     REGCLASS,
```

Comment:
not a fan of this name cause this will get ugly if we add support for more object types, would prefer either just hypertable or a generic name like object or relation

Reply:
I am OK with relation, but FWIW, this was there before and I am just making things consistent.